lib/VeresOne.js

```diff
@@ -24,7 +24,7 @@ class VeresOne {
    * @param [options.client] {WebLedgerClient}
    */
   constructor(options = {}) {
-    this.ledger = 'veres';
+    this.methodId = 'v1';
```

Comment:
It's a little strange to me that this isn't just `this.method`. If this is only used internally, I'd recommend changing to that if there was no strong reason not to.

Reply:
Sure thing, will rename.